Formula/kops.rb

```diff
@@ -25,11 +25,11 @@ class Kops < Formula
 
   def install
     ENV["VERSION"] = version unless build.head?
-    ENV["GOPATH"] = buildpath
+    ENV["GOBIN"] = buildpath
     kopspath = buildpath/"src/k8s.io/kops"
     kopspath.install Dir["*"]
     system "make", "-C", kopspath
-    bin.install "bin/kops"
+    bin.install "kops"
```

Comment:
What does this fix exactly?

Reply:
As far as I can tell, kops now builds based on `$GOBIN` path which is why `GOBIN` was subbed for `GOPATH`. That change caused kops to build into the `buildpath` directory which no longer includes being in a `/bin` folder during the build process.  

See: https://github.com/kubernetes/kops/commit/5b90e580d099b66d4742df3dbb2e220e7b514a8a#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R99 for the main change that I think affected this.

I'm open to suggestions if you know better solutions!

Reply:
Their `Makefile` just seems to call `go build`. Can we just do that directly and align this formulae with the many other formulae that build using `go`?

Reply:
Fair enough, I believe we couldn't historically for some reason but I've modified the PR to a simple go build. Let me know if you if you have any follow ups to those changes, thanks!